[0m18:54:47.391296 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255FA8CAA20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255FCE2E600>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255FCE2CC80>]}


============================== 18:54:47.398304 | 1625f914-c860-4786-9b5c-f9866f1578b7 ==============================
[0m18:54:47.398304 [info ] [MainThread]: Running with dbt=1.10.13
[0m18:54:47.398304 [debug] [MainThread]: running dbt with arguments {'write_json': 'True', 'profiles_dir': 'C:\\Users\\mbdal\\.dbt', 'log_format': 'default', 'printer_width': '80', 'invocation_command': 'dbt init stock_dbt', 'warn_error': 'None', 'static_parser': 'True', 'fail_fast': 'False', 'indirect_selection': 'eager', 'send_anonymous_usage_stats': 'True', 'log_path': 'logs', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'partial_parse': 'True', 'version_check': 'True', 'use_experimental_parser': 'False', 'quiet': 'False', 'introspect': 'True', 'empty': 'None', 'log_cache_events': 'False', 'debug': 'False', 'no_print': 'None', 'use_colors': 'True', 'cache_selected_only': 'False', 'target_path': 'None'}
[0m18:54:47.421590 [debug] [MainThread]: Starter project path: C:\Users\mbdal\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m18:54:47.458716 [info ] [MainThread]: 
Your new dbt project "stock_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m18:54:47.459296 [info ] [MainThread]: Setting up your profile.
[0m19:03:30.950904 [info ] [MainThread]: Profile stock_dbt written to C:\Users\mbdal\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m19:03:30.953707 [debug] [MainThread]: Command `dbt init` succeeded at 19:03:30.953707 after 523.71 seconds
[0m19:03:30.954748 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255FAD44B60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002558F32A240>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255FA7C3410>]}
[0m19:03:30.955762 [debug] [MainThread]: Flushing usage events
[0m19:03:32.176402 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:15:06.532893 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BBDA1B6510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BBDD3C3B30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BBDD3C0410>]}


============================== 19:15:06.537917 | d792540f-d7d6-46ed-bb7b-e52bc982e429 ==============================
[0m19:15:06.537917 [info ] [MainThread]: Running with dbt=1.10.13
[0m19:15:06.538576 [debug] [MainThread]: running dbt with arguments {'target_path': 'None', 'invocation_command': 'dbt test', 'cache_selected_only': 'False', 'no_print': 'None', 'log_format': 'default', 'warn_error': 'None', 'static_parser': 'True', 'partial_parse': 'True', 'indirect_selection': 'eager', 'send_anonymous_usage_stats': 'True', 'fail_fast': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'log_path': 'logs', 'use_experimental_parser': 'False', 'version_check': 'True', 'quiet': 'False', 'introspect': 'True', 'empty': 'None', 'log_cache_events': 'False', 'debug': 'False', 'profiles_dir': 'C:\\Users\\mbdal\\.dbt', 'use_colors': 'True', 'printer_width': '80', 'write_json': 'True'}
[0m19:15:06.538576 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\Python\Polygon.api\stock-api-pipeline\astro_airflow\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m19:15:06.539847 [debug] [MainThread]: Command `dbt test` failed at 19:15:06.539847 after 0.17 seconds
[0m19:15:06.540864 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BBDCA74680>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BBDD767860>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BBDD32F680>]}
[0m19:15:06.540864 [debug] [MainThread]: Flushing usage events
[0m19:15:07.413450 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:15:15.744715 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7EBC0F770>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7E9A86510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7EC4C3D70>]}


============================== 19:15:15.751077 | 1f0f6405-63ff-466a-b71a-d893e0f7aab5 ==============================
[0m19:15:15.751077 [info ] [MainThread]: Running with dbt=1.10.13
[0m19:15:15.751608 [debug] [MainThread]: running dbt with arguments {'target_path': 'None', 'log_format': 'default', 'profiles_dir': 'C:\\Users\\mbdal\\.dbt', 'cache_selected_only': 'False', 'invocation_command': 'dbt test', 'warn_error': 'None', 'indirect_selection': 'eager', 'partial_parse': 'True', 'fail_fast': 'False', 'send_anonymous_usage_stats': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'log_path': 'logs', 'version_check': 'True', 'use_experimental_parser': 'False', 'introspect': 'True', 'quiet': 'False', 'empty': 'None', 'log_cache_events': 'False', 'debug': 'False', 'no_print': 'None', 'use_colors': 'True', 'printer_width': '80', 'write_json': 'True'}
[0m19:15:15.752125 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\Python\Polygon.api\stock-api-pipeline\astro_airflow\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m19:15:15.753183 [debug] [MainThread]: Command `dbt test` failed at 19:15:15.753183 after 0.12 seconds
[0m19:15:15.753734 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7EBA22900>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7EA0C5070>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7ECCCB980>]}
[0m19:15:15.754282 [debug] [MainThread]: Flushing usage events
[0m19:15:16.509149 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:24:32.356734 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000175CA26E720>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000175CCE797C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000175CCE796D0>]}


============================== 19:24:32.356734 | ea8ac1df-16ef-4edb-bb54-5db03a68da09 ==============================
[0m19:24:32.356734 [info ] [MainThread]: Running with dbt=1.10.13
[0m19:24:32.356734 [debug] [MainThread]: running dbt with arguments {'use_colors': 'True', 'no_print': 'None', 'write_json': 'True', 'log_format': 'default', 'printer_width': '80', 'warn_error': 'None', 'static_parser': 'True', 'partial_parse': 'True', 'indirect_selection': 'eager', 'send_anonymous_usage_stats': 'True', 'fail_fast': 'False', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'log_path': 'logs', 'version_check': 'True', 'use_experimental_parser': 'False', 'introspect': 'True', 'quiet': 'False', 'empty': 'None', 'log_cache_events': 'False', 'debug': 'False', 'profiles_dir': 'C:\\Users\\mbdal\\.dbt', 'invocation_command': 'dbt test', 'cache_selected_only': 'False', 'target_path': 'None'}
[0m19:24:32.368810 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\Python\Polygon.api\stock-api-pipeline\astro_airflow\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m19:24:32.368810 [debug] [MainThread]: Command `dbt test` failed at 19:24:32.368810 after 0.15 seconds
[0m19:24:32.368810 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000175CCF0C320>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000175CA26E720>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000175CCFECE30>]}
[0m19:24:32.368810 [debug] [MainThread]: Flushing usage events
[0m19:24:33.255240 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m21:34:34.013545 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BF23E5ABD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BF23D536B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BF26F49D90>]}


============================== 21:34:34.013545 | 3113b2e7-98ef-4067-92ad-a9292cbb38cb ==============================
[0m21:34:34.013545 [info ] [MainThread]: Running with dbt=1.10.13
[0m21:34:34.013545 [debug] [MainThread]: running dbt with arguments {'target_path': 'None', 'profiles_dir': 'C:\\Users\\mbdal\\.dbt', 'no_print': 'None', 'cache_selected_only': 'False', 'invocation_command': 'dbt test', 'warn_error': 'None', 'static_parser': 'True', 'fail_fast': 'False', 'partial_parse': 'True', 'send_anonymous_usage_stats': 'True', 'log_path': 'logs', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'indirect_selection': 'eager', 'use_experimental_parser': 'False', 'version_check': 'True', 'introspect': 'True', 'quiet': 'False', 'empty': 'None', 'log_cache_events': 'False', 'debug': 'False', 'log_format': 'default', 'use_colors': 'True', 'printer_width': '80', 'write_json': 'True'}
[0m21:34:34.013545 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\Python\Polygon.api\stock-api-pipeline\astro_airflow\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:34:34.022571 [debug] [MainThread]: Command `dbt test` failed at 21:34:34.022571 after 0.16 seconds
[0m21:34:34.022571 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BF23FE7830>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BF23D536B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001BF27295220>]}
[0m21:34:34.022571 [debug] [MainThread]: Flushing usage events
[0m21:34:34.952889 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m21:48:57.275734 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020F65B2FB00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020F6903E8A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020F69924D70>]}


============================== 21:48:57.279366 | 72f0ab2f-d3b9-460d-b8c4-1a999128e168 ==============================
[0m21:48:57.279366 [info ] [MainThread]: Running with dbt=1.10.13
[0m21:48:57.279366 [debug] [MainThread]: running dbt with arguments {'use_colors': 'True', 'cache_selected_only': 'False', 'no_print': 'None', 'target_path': 'None', 'profiles_dir': 'C:\\Users\\mbdal\\.dbt', 'warn_error': 'None', 'static_parser': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'send_anonymous_usage_stats': 'True', 'partial_parse': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'indirect_selection': 'eager', 'use_experimental_parser': 'False', 'version_check': 'True', 'introspect': 'True', 'quiet': 'False', 'empty': 'None', 'log_cache_events': 'False', 'debug': 'False', 'log_format': 'default', 'invocation_command': 'dbt debug', 'printer_width': '80', 'write_json': 'True'}
[0m21:48:57.316867 [info ] [MainThread]: dbt version: 1.10.13
[0m21:48:57.316867 [info ] [MainThread]: python version: 3.12.5
[0m21:48:57.316867 [info ] [MainThread]: python path: C:\Users\mbdal\AppData\Local\Programs\Python\Python312\python.exe
[0m21:48:57.316867 [info ] [MainThread]: os info: Windows-11-10.0.22631-SP0
[0m21:48:57.326590 [info ] [MainThread]: Error importing adapter: No module named 'dbt.adapters.postgres'
[0m21:48:59.226178 [info ] [MainThread]: Using profiles dir at C:\Users\mbdal\.dbt
[0m21:48:59.227206 [info ] [MainThread]: Using profiles.yml file at C:\Users\mbdal\.dbt\profiles.yml
[0m21:48:59.228195 [info ] [MainThread]: Using dbt_project.yml file at D:\Python\Polygon.api\stock-api-pipeline\astro_airflow\dbt_project.yml
[0m21:48:59.229199 [info ] [MainThread]: Configuration:
[0m21:48:59.229938 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m21:48:59.229938 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m21:48:59.231535 [info ] [MainThread]: Required dependencies:
[0m21:48:59.231846 [debug] [MainThread]: Executing "git --help"
[0m21:48:59.280544 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m21:48:59.281545 [debug] [MainThread]: STDERR: "b''"
[0m21:48:59.281833 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m21:48:59.281833 [info ] [MainThread]: Connection test skipped since no profile was found
[0m21:48:59.283842 [info ] [MainThread]: [31m2 checks failed:[0m
[0m21:48:59.283842 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  Credentials in profile "dbt_first", target "dev" invalid: Runtime Error
    Could not find adapter type postgres!


[0m21:48:59.283842 [info ] [MainThread]: Project loading failed for the following reason:
 project path <D:\Python\Polygon.api\stock-api-pipeline\astro_airflow\dbt_project.yml> not found

[0m21:48:59.283842 [debug] [MainThread]: Command `dbt debug` failed at 21:48:59.283842 after 2.17 seconds
[0m21:48:59.283842 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020F6C366900>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020F7E37CA40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020F7C0AC860>]}
[0m21:48:59.283842 [debug] [MainThread]: Flushing usage events
[0m21:49:00.207676 [debug] [MainThread]: An error was encountered while trying to flush usage events
